src/index.html

```diff
@@ -9,10 +9,72 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
Styles should be included with a `.css` extension, not `.scss`. SCSS files are preprocessed and not understood by browsers directly. You need to link the compiled CSS file instead.